linalg/hypre.hpp

```diff
@@ -141,8 +141,10 @@ class HypreParVector : public Vector
        allocated in the memory location HYPRE_MEMORY_DEVICE. */
    HypreParVector(MPI_Comm comm, HYPRE_BigInt glob_size, double *data_,
                   HYPRE_BigInt *col, bool is_device_ptr = false);
-   /// Creates vector compatible with y
+   /// Creates vector compatible with y containing a deep copy of the local data
```

Comment:
Maybe just this is clearer?
```suggestion
/// Creates a deep copy of @a y
```